kotlin/mobilesdkrs/build.gradle

```diff
@@ -119,15 +119,15 @@ publishing {
         mavenLocal()
     }
     publications {
-        // debug(MavenPublication) {
-        //     groupId = 'com.spruceid.mobile.sdk.rs'
-        //     artifactId = "mobilesdkrs"
-        //     version = System.getenv("VERSION")
-
-        //     afterEvaluate {
-        //         from components.release
-        //     }
-        // }
+        debug(MavenPublication) {
+            groupId = 'com.spruceid.mobile.sdk.rs'
+            artifactId = "mobilesdkrs"
+            version = System.getenv("VERSION")
+
+            afterEvaluate {
+                from components.release
+            }
+        }
```

Comment:
NOTE: I am adding this back because it is needed for releasing to `mavenLocal()` repository.

Reply:
Just for reference, for this to be able to stay in the repo, the CI needs to be updated use individual `publish` calls, that is, `./gradlew publishReleasePublicationToGitHubPackagesRepository` and `./gradlew publishReleasePublicationToNmcpReleaseRepository` instead of `./gradlew publish`.

Reply:
Ah, this is good to know! :pray:

Reply:
Does `publishReleasePublicationToCentralPortal` provide support for Nmcp? 

I didn't add `publishReleasePublicationToNmcpReleaseRepository` step to the release, but I did replace `publish` with `publishReleasePublicationToGitHubPackagesRepository ` in [77e35df](https://github.com/spruceid/mobile-sdk-rs/pull/58/commits/77e35dffda6b6845578f9bdf6b9dde281df4327a)